### PR TITLE
Cache queries

### PR DESCRIPTION
**Pull request by dtrucs**
Put common dictionaries (themes, cities, sources, informationDesk, labels)  in the server cache and use it for search/details/suggestions pages

### CODE REVIEW

File: frontend/src/components/Map/hooks/usePopupResult.ts [hunk 1]
```diff
@@ -5,8 +5,11 @@ import { PopupResult } from 'modules/trekResult/interface';
 import { getTouristicContentPopupResult } from 'modules/touristicContent/connector';
 import { useRouter } from 'next/router';
 import { getDefaultLanguage } from 'modules/header/utills';
-import { getOutdoorSitePopupResult } from '../../../modules/outdoorSite/connector';
+import { CommonDictionaries } from 'modules/dictionaries/interface';
+import { getCommonDictionaries } from 'modules/dictionaries/connector';
+import { ONE_DAY } from 'services/constants/staleTime';
```
Comment (Bo-Duke) [inline frontend/src/components/Map/hooks/usePopupResult.ts:10]: Maybe create a HALF_DAY constant? It's used in a LOT of files

File: frontend/src/components/pages/touristicContent/useTouristicContent.tsx [hunk 2]
```diff
@@ -20,11 +22,25 @@ export const useTouristicContent = (
...
+    {
+      onError: async error => {
+        if (isRessourceMissing(error)) {
+          await router.push(routes.HOME);
+        }
+      },
+      staleTime: ONE_DAY / 2,
+    },
+  );
+
```
Comment (Bo-Duke) [inline frontend/src/components/pages/touristicContent/useTouristicContent.tsx:38]: Lots of duplicate code, can't this logic be abstracted in a file?

Comment (dtrucs) [reply frontend/src/components/pages/touristicContent/useTouristicContent.tsx]: Good point, i DRY this part with a dedicated file :+1:

File: frontend/src/components/Map/hooks/usePopupResult.ts (hunk 1 above)
Comment (dtrucs) [reply frontend/src/components/Map/hooks/usePopupResult.ts]: With the DRY, it's now call once. Do you think it's still necessary?

Comment (Bo-Duke) [reply frontend/src/components/Map/hooks/usePopupResult.ts]: Nope, perfect this way 👌

**Review by Bo-Duke — APPROVED** [review]

**Comment by camillemonchicourt** [comment]
A répercuter dans https://github.com/GeotrekCE/Geotrek-rando-v3/blob/main/docs/knowledge/caching.md ?

**Pull request merged**